TESTING.md

```
@@ -0,0 +1,171 @@
+```bash
+# Check if database is running
+docker compose ps test-db
+
+# View database logs
+docker compose logs test-db
+
+# Restart database
+docker compose restart test-db
+```
+
+### Authentication Issues
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)